src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx

```diff
@@ -106,21 +156,22 @@ Set the [thresholds](/docs/alerts/new-relic-alerts-beta/configuring-alert-polici
   </tbody>
 </table>
 
-In most of our products (except Infrastructure), the color-coded [health status indicator](https://docs.newrelic.com/docs/accounts-partnerships/education/getting-started-new-relic/glossary#health-status) in the user interface changes as the alerting threshold escalates or returns to normal. This allows you to monitor a situation through our UI before a critical threshold passes, without needing to receive specific notifications about it.
+In most of our products (except Infrastructure), the color-coded [health status indicator](/docs/using-new-relic/welcome-new-relic/get-started/glossary/#health-status) in the user interface changes as the alerting threshold escalates or returns to normal. This allows you to monitor a situation through our UI before a critical threshold passes, without needing to receive specific notifications about it.
+ 
+There are two violation thresholds: critical (red) and warning (yellow). Define these thresholds with different criteria keeping in mind the suggestions above. 
 
-For example, you can define a critical (red) threshold that notifies you when the error percentage for your app is above 10 percent at least once in any five minute period. You can also define an optional warning (yellow) threshold with different criteria.
-
-## Select notification channels [#channels]
+<Callout variant='important'>
+Warning violations do **not** open incidents. A critical violation can open incidents, but you must define that decision through your [incident preferences](/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices/#incident-practices).
+</Callout>
 
-You can create [notification channels](/docs/alerts/new-relic-alerts-beta/managing-notification-channels/notification-channels-controlling-where-send-alerts) first and then assign policies to them. You can also create policies first and then assign notification channels to them. This flexibility allows you to tailor who gets notified, using the method that is most useful to them.
+## Muting roles [#mute-practices]
 
-For example, you could:
+Mute alerts during routined events such as maintenance or planned downtime.
 
-* Identify your operations team's Slack channel as a general level of alerting, and use the on-call PagerDuty contact as an after-hours or escalated level of alerting.
-* Create webhooks with customized messages for a variety of situations or personnel.
+You can also silence another aspect of Alerts (like a policy, a specific entity, a condition) when needed. Incidents can still be opened, but you will not be notified.
```

Comment:
I think you should rewrite this first sentence to remove mention of alerts and not use a parenthetical.

Reply:
I prefer "won't" to "will not"